driver/src/main/scala/com/stratio/sparkta/driver/SparktaClusterJob.scala

```diff
@@ -44,30 +47,35 @@ object SparktaClusterJob extends SparktaSerializer {
     if (checkArgs(args)) {
       Try {
         initSparktaConfig(args(4), args(3))
```

Comment:
I had the same code that Sergio says, but as the method is to big and clear. If I change it as Alvaro says and I can remove the magic numbers.

I make one small refactor and submmit changes